python/dgl/dataloading/base.py

```diff
@@ -313,6 +313,21 @@ def _find_exclude_eids(g, exclude_mode, eids, **kwargs):
     else:
         raise ValueError("unsupported mode {}".format(exclude_mode))
 
+class SpotTarget(object):
```

Comment:
Would you mind adding a docstring on this?  Since it will be user-facing, we expect it to at least have the following sections:
```
class SpotTarget(object):
    """[Describe what this class does.  Even better, share a link to your paper.]

    Parameters
    ----------
    [Describe what each parameter does, i.e. g and degree_threshold]

    Examples
    --------
    [Show an example of how to use this class]
    """
    ...
```